assets/base.css

```diff
@@ -1191,37 +1191,71 @@ deferred-media {
   position: relative;
 }
 
+.share-button__button {
+  font-size: 1.4rem;
+  display: flex;
+  align-items: baseline;
+  color: rgba(var(--color-link),var(--alpha-link));
+  margin-left: 0;
+  padding-left: 0;
+
+}
+
+.share-button__button, .share-button__fallback button {
+  cursor: pointer;
+  background: var(--color-background);
+  border: none;
+}
+
 .share-button__button .icon-share {
   margin-right: 1rem;
 }
 
 .share-button__fallback {
```

Comment:
I think so 🤷  it was not changed within this PR. But seeing what Olivia mentioned it will have a max-width of `310px` (https://github.com/Shopify/dawn/pull/349#issuecomment-895503605)